packages/common/string.js

```diff
@@ -136,17 +151,19 @@ const isValidHostname = flow(toLower, matches(hostnameRegex));
  * @param {string} x - the string to test
  * @returns {boolean}
  *
+ * @deprecated
+ *
  * @alias module:string
  */
 const isNonEmptyString = (x) => isString(x) && x.length > 0;
 
 module.exports = {
-  globalReplace,
-  isNonEmptyString,
-  isValidHostname,
-  match,
-  matches,
-  replace,
-  toLower,
-  toUpper
+  globalReplace: deprecate(globalReplace, '@cumulus/common/string.globalReplace() is deprecated after version 1.21.0 and will be removed in a future release. Use lodash/replace instead.'),
+  isNonEmptyString: deprecate(isNonEmptyString, '@cumulus/common/string.isNonEmptyString() is deprecated after version 1.21.0 and will be removed in a future release.'),
+  isValidHostname: deprecate(isValidHostname, '@cumulus/common/string.isValidHostname() is deprecated after version 1.21.0 and will be removed in a future release. Use is-valid-hostname instead.'),
+  match: deprecate(replace, '@cumulus/common/string.match() is deprecated after version 1.21.0 and will be removed in a future release.'),
+  matches: deprecate(replace, '@cumulus/common/string.matches() is deprecated after version 1.21.0 and will be removed in a future release.'),
+  replace: deprecate(replace, '@cumulus/common/string.replace() is deprecated after version 1.21.0 and will be removed in a future release. Use lodash/fp/replace instead.'),
+  toLower: deprecate(toLower, '@cumulus/common/string.toLower() is deprecated after version 1.21.0 and will be removed in a future release. Use lodash/fp/toLower instead.'),
```

Comment:
IS `toLower` in `lodash/fp` package? I didn't see it.

Reply:
That was a copy / paste error, which I've fixed.